requirements.txt

```diff
@@ -1,11 +1,11 @@
-aiohttp>3.5.3
-backoff==1.10.0
-nest_asyncio
-pyjwt==2.4.0
-pandas
-funcy
-lenses
-toolz
-pydantic
-python-pmap
-fastparquet
+aiohttp>3.5.3,<4.0.0
+backoff>=1.10.0,<2.0.0
+nest_asyncio>=1.5.6,<2.0.0
+pyjwt>=2.4.0,<3.0.0
+pandas>=1.3.5,<2.0.0
```

Comment:
I understand this is where it is now. but hopefully it could work with pandas v2.0.0+ soon which is used by the platform jupyter notebook feature (before rolling back)

Reply:
Thanks Shawn I agree. That is a good next step.